src/robot/parsing/model.py

```diff
@@ -595,20 +595,33 @@ def __iter__(self):
 
 
 class ForLoop(_WithSteps):
+    """
+    Properties:
+
+    - flavor: The value of the 'IN' item, with spaces removed.
+              Typically 'IN', 'INRANGE', 'INZIP', 'INEUMERATE', etc.(?)
+    - range:  True if flavor is 'INRANGE'
+    - vars:   Variables set per-iteration by this loop
+    - items:  Items to loop over (or otherwise parameters, like with INRANGE)
+    - comment: a comment, or None.
+    - steps:  A list of steps in the loop.
+    """
```

Comment:
I'll look into this after adding more tests and before updating the documentation.